### ISSUE TITLE
Small bug when hiding the key?

### ISSUE THREAD

**Issue by teodorescuserban** [darkweak/souin#564]
Hello,

I am using souin as caddy plugin (1.7.2)
I noticed that when I am trying to hide the key, the key is hidden, but the `key=` is not.

```
cache-status: test; hit; ttl=2; key=; detail=REDIS
```
vs
```
cache-status: pilula; hit; ttl=4; key=/some/key ; detail=REDIS
```

Versions:
```
/etc/caddy # caddy version
v2.8.4 h1:q3pe0wpBj1OcHFZ3n/1nl4V4bxBrYoSoab7rL9BMYNk=
/etc/caddy # caddy build-info | grep souin
dep     github.com/darkweak/souin       v1.7.2  h1:i9t/fdCgvBuKM7NNYcqnCQCmFlk9nEF91ltuksfFyjs=
dep     github.com/darkweak/souin/plugins/caddy v1.7.2  h1:EVYHOcRsr3XnM45RiGKATogk3qy1/cH8rVR8p+BDe14=
```

**Comment by darkweak** [comment]
Hey @teodorescuserban this is the desired behavior.

**Comment by darkweak** [comment]
@teodorescuserban the commit `c816c2ba5f81a33c016cf806264b5ad3474dda5b` contains the fix, can you try it?
```
xcaddy build --with github.com/darkweak/souin/plugins/caddy@c816c2ba5f81a33c016cf806264b5ad3474dda5b --with github.com/darkweak/souin@c816c2ba5f81a33c016cf806264b5ad3474dda5b
```

**Comment by teodorescuserban** [comment]
> Hey @teodorescuserban this is the desired behavior.

Would be better to have `key=*` so it's obvious that it is hidden; just `key=` makes you think there's an issue and something is incomplete.

But if this is the intended behaviour, I guess this issue can be closed 😄

**Comment by darkweak** [comment]
I didn't think about using stars to tell the users it's hidden. 👍

**Comment by teodorescuserban** [comment]
Leaving it open, please close it anytime you want, whether you add some `***` or not. :)

**Comment by darkweak** [comment]
@teodorescuserban implemented this behavior (`*****`) when key is hidden.

**Comment by darkweak** [comment]
Available in v1.7.5 🙂
